<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginView</class>
 <widget class="QWidget" name="LoginView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OctoVault VPN</string>
  </property>
  <widget class="QLabel" name="lblBackgroundImg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>500</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>500</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../assets/assets.qrc">:/img/images/img_background.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblLogo">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>50</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../assets/assets.qrc">:/img/images/logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="usernameHolder" native="true">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>210</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background:transparent;
border: 1px solid #36353a;
border-radius: 10px;
padding-left: 10px;
}

QWidget:hover{
background:transparent;
border: 1px solid #FACA7F;
border-radius: 10px;
padding-left: 10px;
}</string>
   </property>
   <widget class="QLabel" name="lblProfile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>20</width>
      <height>10</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>10</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>10</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border-color: transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/assets.qrc">:/img/images/icon_profile.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="editEmail">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>10</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background:transparent;
padding-left:0px;
color: #36353a;
border-color: transparent;
}

QLineEdit:hover{
background:transparent;
padding-left:0px;
color: #FACA7F;
border-color: transparent;
}</string>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="passHolder" native="true">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>260</y>
     <width>250</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>40</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background:transparent;
border: 1px solid #36353a;
border-radius: 10px;
padding-left: 10px;
}

QWidget:hover{
background:transparent;
border: 1px solid #FACA7F;
border-radius: 10px;
padding-left: 10px;
}</string>
   </property>
   <widget class="QLabel" name="lblPass">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>20</width>
      <height>10</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>20</width>
      <height>10</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>20</width>
      <height>10</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
border-color: transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="../assets/assets.qrc">:/img/images/icon_lock.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnShowHide">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-color: transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/img/images/icon_show.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPass">
    <property name="geometry">
     <rect>
      <x>23</x>
      <y>10</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
padding-left:0px;
color: #36353a;
border-color: transparent;
}

QLineEdit:hover{
background:transparent;
padding-left:0px;
color: #FACA7F;
border-color: transparent;
}</string>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnForget">
   <property name="geometry">
    <rect>
     <x>325</x>
     <y>320</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
text-decoration: underline;
color: #FACA7F;
background-color: transparent;
}</string>
   </property>
   <property name="text">
    <string>Forgot Your Password?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSignUp">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>450</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color: #FFFFFF;
background-color: transparent;
}</string>
   </property>
   <property name="text">
    <string>Don't have an account? Sign up!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLogin">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>410</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>250</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>250</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#FACA7F;
border-radius: 15px;
color:#000000;
}</string>
   </property>
   <property name="text">
    <string>Log In</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSpinner">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>360</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblIntroOne">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>200</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>Welcome back!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblIntroTwo">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>300</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:#FACA7F;
}</string>
   </property>
   <property name="text">
    <string>We are happy to see you again</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblLogoTwo">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>50</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../assets/assets.qrc">:/img/images/logo_text.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChangeLogo">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:transparent;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
